### ISSUE TITLE
Anyone wants to check their 'Berichten' on Mister? 

### ISSUE THREAD

**Issue by KeizerDev** [KeizerDev/Mister#16]
I want to ask the folks if they want to read their `Berichten` in Mister?

**Comment by Vogeltak** [comment]
Is it pushed to the play store yet? Or should I install an apk?

**Comment by KeizerDev** [comment]
No, I actually still haven't made it... This is just to discuss if it is worth to make it!

**Comment by Vogeltak** [comment]
Aha ok. Well, my messages get forwarded to my email and I read them there. If you implement it it should not be read only, figure out a way to easily send messages. So for the sake of a complete package, I would implement it.

**Comment by KeizerDev** [comment]
Hm, my magister doens't use the Magister mail, we use gmail... So I have a friend from another school who used Magister mail, I can use his account to develop this. I think it would not be a big to problem to figure out how to mail using the Magister api.

**Comment by Vogeltak** [comment]
As you know I had some trouble with implementing a mail spammer. Maybe [this](https://github.com/Vogeltak/magister-message-spam/pull/2) is still of any help for you. I don't really like the way they implemented mail in Magister. I would be really interested in keeping up with your developments regarding the implementation of mail. If you get things working could you explain some of it to me?

**Comment by KeizerDev** [comment]
Yeah for sure!

**Comment by Vogeltak** [comment]
Great!

**Comment by KeizerDev** [comment]
LMAO! I'VE FIGURE IT OUT!

**Comment by KeizerDev** [comment]
We can chat about it on XMPP?!

**Comment by raqbit** [comment]
Yeah :D Would Love this feature. Also, a bit offtopic, but can't you implement Grade listing for periods? For example, our school has 3 periods, in the origional magister app(site) you get a dropdown menu (at the grades tab) to select the period, then you can find the grades for that period. In Mister, they all show up in the same list. quite confusing

**Comment by KeizerDev** [comment]
@JustRamon I've found a way to send mails through the Magister api only, so I'm gonna implement the `Berichten` soon I think. About the grades, It's pretty hard to implement this good and make it work for every school that is using Magister, so I think I'm gonna replace it with a webview till I've found a good way to implement it native.

**Comment by raqbit** [comment]
Ok! Im ok with that :P It's your project!
